tests/integration/models/test_inbox.py

```diff
@@ -45,6 +45,14 @@ def test_comment_reply__refresh(self, _):
             comment.refresh()
             assert saved_id == comment.id
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_mark_all_read(self, _):
+        self.reddit.read_only = False
+        with self.use_cassette():
+            self.reddit.inbox.mark_unread(list(self.reddit.inbox.all(limit=2)))
```

Comment:
Do you think we should check that there are actually unread messages? I'm fine with leaving it as is if you don't think it's necessary.

Reply:
When I tested this with invalid `filter_types` values, I found that it would return a 202 even if it didn't mark the messages as read. It's not strictly necessary.